src/Integration/MonologV2/CloudWatch/src/LogHandler.php

```diff
@@ -0,0 +1,314 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AsyncAws\MonologV2\CloudWatch;
```

Comment:
Could we support both v1 and v2 is monolog? Are the Handers very different? 
I don’t remember them being very different... but I might be wrong.

Reply:
the v2 handler adds return types to all interface methods, so I don't think it can be the same file?

Reply:
Symfony use this trick:

```php
// CloudWatchHandlerV1.php
class CloudWatchHandlerV1 {
  public function foo(){}
}

// CloudWatchHandlerV2.php
class CloudWatchHandlerV2 {
  public function foo(): void {}
}

// CloudWatchHandler.php
if (interface_exists(InterfaceThatExistsOnlyInV2::class)) {
  class CloudWatchHandler extends CloudWatchHandlerV2{}
} else {
  class CloudWatchHandler extends CloudWatchHandlerV1{}
}

Reply:
But I'm also fine if we only support v2.

Reply:
If it is only return types, then it is fine: https://3v4l.org/qnojm

Reply:
It seems like I was wrong, although there is an interface the change is also applied on the AbstractProcessingHandler class which should be used as base class, so the trick in https://3v4l.org/qnojm is not applicable

I'll try the Symfony trick

Reply:
Monolog v1 is still massively popular. If we can support both with this small trick then I would be happy.

Reply:
I tested the same class with Monolog v1 and v2 and it seems to work without modification

https://3v4l.org/vhQ3D

Not sure why maxbanton/cwh was tagged 2.0 with the upgrade to Monolog v2, that was probably not required